src/internal/packager/sbom/catalog.go

```diff
@@ -52,51 +54,70 @@ var componentPrefix = "zarf-component-"
 
 // Catalog catalogs the given components and images to create an SBOM.
 func Catalog(ctx context.Context, componentSBOMs map[string]*layout.ComponentSBOM, imageList []transform.Image, paths *layout.PackagePaths) error {
+	l := logger.From(ctx)
 	imageCount := len(imageList)
 	componentCount := len(componentSBOMs)
 	cachePath, err := config.GetAbsCachePath()
 	if err != nil {
 		return err
 	}
 	builder := Builder{
+		// TODO(mkcp): Remove message on logger release
 		spinner:    message.NewProgressSpinner("Creating SBOMs for %d images and %d components with files.", imageCount, componentCount),
 		cachePath:  cachePath,
 		imagesPath: paths.Images.Base,
 		outputDir:  paths.SBOMs.Path,
 	}
 	defer builder.spinner.Stop()
+	l.Info("creating SBOMs for images and components with files", "images", imageCount, "components", componentCount)
```

Comment:
Thoughts on deleting this message since it may be redundant alongside the other "creating SBOMs for images" and "creating SBOMs for components" messages

Reply:
Yeah that's fine - it did feel a bit redundant when I was porting over the spinner messages.